_includes/manuals/2.1/2_quickstart_guide.md

```diff
@@ -1,12 +1,14 @@
 
 The Foreman installer is a collection of Puppet modules that installs everything required for a full working Foreman setup.  It uses native OS packaging (e.g. RPM and .deb packages) and adds necessary configuration for the complete installation.
 
-Components include the Foreman web UI, Smart Proxy, Passenger, a Puppet master (either Puppet Server or under Passenger), and optionally TFTP, DNS and DHCP servers.  It is configurable and the Puppet modules can be read or run in "no-op" mode to see what changes it will make.
+Components include the Foreman web UI, Smart Proxy, Puma, a Puppet Server, and optionally TFTP, DNS and DHCP servers.  It is configurable and the Puppet modules can be read or run in "no-op" mode to see what changes it will make.
 
 #### Supported platforms
 * CentOS 7 x86_64
+* CentOS 8 x86_64 (except for Katello plugin)
```

Comment:
Normally we don't list support for plugins. Katello users shouldn't even read this manual since they have their own installation instructions. :-1: for doing this.